src/gameplayv2/resistance/components/RoomComponent.js

```diff
@@ -0,0 +1,27 @@
+//extends component
```

Comment:
Don't follow the constructor of the existing room. It's heavily polluted and got a lot of stuff. 

We've got an excellent opportunity to re-design the interface from scratch so it's your time to shine and show what kind of API you think will be the best. We'll later integrate it with the existing room or refactor it completely.